pmoves/services/hi-rag-gateway-v2/geometry_bus.py

```
@@ -0,0 +1,777 @@
+                    )
+                    anchor_id = cur.fetchone()[0]
+                    spectrum = const.get("spectrum") if isinstance(const.get("spectrum"), list) else None
+                    radial = const.get("radial_minmax") if isinstance(const.get("radial_minmax"), list) else [None, None]
+                    summary = const.get("summary")
+                    # insert constellation (preserve CGP/meta provenance if present)
+                    const_meta = {}
+                    try:
+                        if isinstance(const.get("meta"), dict):
+                            const_meta.update(const.get("meta") or {})
+                    except Exception:
+                        pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)